policy/org.freedesktop.fwupd.policy.in

```diff
@@ -207,4 +207,14 @@
     <annotate key="org.freedesktop.policykit.imply">org.freedesktop.fwupd.get-bios-settings</annotate>
   </action>
 
+  <action id="org.freedesktop.fwupd.repair">
+    <description>Repair kernel parameter</description>
```

Comment:
I don't think it's a kernel parameter it's repairing - it's BIOS settings isn't it?

Reply:
My bad, I'll make a new description, for example: "Repair or revert a failed security attribute."